java/src/test/java/org/rocksdb/SstFileReaderTest.java

```diff
@@ -219,4 +219,138 @@ public void readSstFile() throws RocksDBException, IOException {
       assertThat(iterator.value()).isEqualTo("value1".getBytes());
     }
   }
+
+  private void assertInternalKey(final Options options, ParsedEntryInfo parsedEntryInfo,
```

Comment:
Parameters should be marked final. (and methods below)

Reply:
done